Slim/Control/Queries.pm

```diff
@@ -4648,14 +4648,15 @@ sub worksQuery {
 	my $roleID        = $request->getParam('role_id');
 	my $genreID       = $request->getParam('genre_id');
 	my $year          = $request->getParam('year');
+	my $workID        = $request->getParam('work_id');
 
 	# get them all by default
 	my $where = {};
 	my $w   = ["tracks.work IS NOT NULL"];
 	my $p   = [];
 
 	my $columns = "works.title, works.id, composer.name, composer.id, composer.namesort, works.titlesort, GROUP_CONCAT(DISTINCT albums.artwork), GROUP_CONCAT(DISTINCT albums.id)";
-	my $groupBy = "works.title, works.id, composer.name, composer.id, composer.namesort, works.titlesort";
+	my $groupBy = "tracks.work, composer.id";
```

Comment:
What's the reason you make this a variable? From what I can tell it's only ever used in one place, and never modified.

Reply:
Not sure, maybe a previous version did change it but I can't think why. I'll fix.